proc_parent.go

```diff
@@ -478,10 +479,21 @@ func (mp *parent) overwriteBinary(tmpBinPath string) error {
 	})
 }
 
+// Generates a file path in the same directory as the binary, using the hash of
+// the absolute binPath as the filename.
+func (mp *parent) touchFilePath() string {
+	hash := fnv.New64a()
+	hash.Write([]byte(mp.binPath))
+	touchFile := fmt.Sprintf("%x-updates.lock", hash.Sum64())
+	return filepath.Join(filepath.Dir(mp.binPath), touchFile)
+}
```

Comment:
What is the purpose of hashing the full path of the binary? Why not append a constant to the bin path?

```go
mp.binPath + "-updates.lock"
```

Reply:
Agreed. It will be clearer to users what it's for, too. Implemented.